src/features/home/Tiles.jsx

```diff
@@ -24,18 +29,31 @@ import {
   selectHasOpkgBinary,
 } from "../device/deviceSlice";
 
+import { selectUpgradable } from "../packages/packagesSlice";
+
 export default function Tiles() {
   const { t } = useTranslation("home");
+  const translation = useTranslation("setup");
```

Comment:
Please duplicate the strings for each screen.
It might need different translation depending on  the context it is being used.

Reply:
I'm not quite sure I've understood correctly - you mean I should add the required strings to a home json instead of accessing it from different group of translations?

Reply:
Yes, exactly. In case that it is just a word that could be used anywhere and context is not important, simply add the string to **common.json** file. Otherwise I would prefer to keep the translations separated on a per page basis - this allows us not to think about if we used it anywhere else and potentially mess up a translation in a different place.

Reply:
Ok, will think about this when doing stuff in the future, I've seen it to be done somewhere in the project in the similar way so that's why I did it this way

Reply:
Yes, this is quite possible. If you see it anywhere else, feel free to fix it there. Might not have been as consequent myself as I wished ;-)